Dockerfile

```
@@ -0,0 +1,14 @@
+FROM python:3 as base
+RUN curl -sSL https://install.python-poetry.org | python3 -
+ENV PATH=$PATH:/root/.local/bin
+WORKDIR /todo_app
+COPY . /todo_app
+RUN poetry install
```

Minor, but worth noting that we might choose to separate out the COPY command into two parts - first copying in the files that specify our dependencies that are directly relevant to `poetry install`, and then everything else, so that in future builds as often as possible we can cache the install step which is typically the slowest part of our builds
```suggestion
COPY pyproject.toml poetry.lock ./
RUN poetry install
COPY . /todo_app
```